keps/sig-auth/4317-pod-certificates/README.md

```diff
@@ -337,6 +303,14 @@ type PodCertificateRequestSpec struct {
 	// signerName indicates the request signer.
 	SignerName string `json:"signerName" protobuf:"bytes,1,opt,name=signerName"`
 
+  // userConfig allow pod authors to pass additional information to
+  // the signer implementation.
+  //
+  // Keys should be namespaced, in the form `<domainName>/<keyname>`.  Signers
+  // should document the keys and values they support.  Signers should deny
+  // requests that contain keys they do not recognize.
```

Comment:
what would be length and characters limits here? Do we ever expect very long keys or values?